observation_portal/observations/models.py

```diff
@@ -38,7 +38,11 @@ def get_expanded_configurations(observation, configurations):
     # If this is a REAL_TIME observation, just return its configuration_status since it doesn't have a configuration
     if observation.request.request_group.observation_type == RequestGroup.REAL_TIME:
         if observation.configuration_statuses.all().count() > 0:
-            return [{'configuration_status': observation.configuration_statuses.first().id}]
+            config_status = observation.configuration_statuses.first()
+            return [{
+                'configuration_status': config_status.id,
+                'state': config_status.state
```

Comment:
This is another unrelated change that adds in the 'state' for the 'configurations' section of REAL_TIME submissions.